var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -27,28 +27,31 @@
 
 class Paraview(Package):
```

Comment:
OK, got it. The problem isn't CMakePackage in itself. (ie, spack install paraview is OK). The problem happens when I try `spack build paraview`. Then it goes crazy and bombs out while resolving hwloc for openmpi.

I'll make the Cmake change